pkg/web/static/openapi.json

```diff
@@ -1921,8 +1920,7 @@
         "type": "object",
         "properties": {
           "amount": {
-            "type": "number",
-            "format": "double"
+            "type": "number"
```

Comment:
Did you change it back by accident?

Reply:
I thought I'd pulled the changes. I'll take another look so that we can get this approved.